apps/build.gradle

```diff
@@ -57,15 +57,17 @@ subprojects {
         delete "${project.parent.buildDir}/resources/main/public_html/apps/${project.name}"
     }
 
-    //copy styleguide or normal app
+    //copy nuxt-ssr, styleguide or normal app
     if(project.name == 'nuxt-ssr') {
```

Comment:
why does nuxt need seperate copy, can't you use copy we use for all other apps?

Reply:
it has a different folder structure